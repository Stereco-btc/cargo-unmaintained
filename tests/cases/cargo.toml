url = "https://github.com/rust-lang/cargo"
rev = "0.74.0"
