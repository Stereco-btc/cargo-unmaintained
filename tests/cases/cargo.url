https://github.com/rust-lang/cargo
