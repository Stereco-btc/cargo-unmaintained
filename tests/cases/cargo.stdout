sized-chunks (https://github.com/bodil/sized-chunks updated [..] days ago)
    bitmaps (requirement: ^2.1.0, version used: 2.1.0, latest: [..])
im-rc (https://github.com/bodil/im-rs updated [..] days ago)
    bitmaps (requirement: ^2, version used: 2.1.0, latest: [..])
    sized-chunks (requirement: ^0.6.4, version used: 0.6.5, latest: [..])
partial_ref_derive (https://github.com/jix/partial_ref updated [..] days ago)
    syn (requirement: ^1.0.40, version used: 1.0.109, latest: [..])
matchers (https://github.com/hawkw/matchers updated [..] days ago)
    regex-automata (requirement: ^0.1, version used: 0.1.10, latest: [..])
serde-value (https://github.com/arcnmx/serde-value updated [..] days ago)
    ordered-float (requirement: ^2.0.0, version used: 2.10.0, latest: [..])
rusty-fork (https://github.com/altsysrq/rusty-fork updated [..] days ago)
    quick-error (requirement: ^1.2, version used: 1.2.3, latest: [..])
anes (https://github.com/zrzka/anes-rs updated [..] days ago)
    bitflags (requirement: ^1.2, version used: 1.3.2, latest: [..])
